modules/support-gather-data.adoc

```diff
@@ -38,6 +38,11 @@ If your cluster is using a restricted network, you must take additional steps. I
 $ oc import-image is/must-gather -n openshift
 ----
 ====
++
+[NOTE]
+====
+Because this command picks a random master by default, the pod might be scheduled to a master that is in `NotReady` and `SchedulingDisabled` state.
```

Comment:
Two things:

* master -> control plane node - two places (since we avoid the term 'master' now when possible)
* that is in -> that is in the ?
* This note doesn't really give the user any indication on whether this is okay if this happens, or if they need to do something if this happens (like re-run the must-gather command?)

Reply:
For the last point, Ithat's very much valid. However, there are multiple actions/work-around that the user can opt for. Re-running the command could be one, other could be to selectively point to the required node using the `--node-name` flag. IMO, it won't be apt to provide work-arounds in the documentation. The SME's ask in the bug also states that the user should be notified that this scenario could happen. I assume the user would have enough expertise to understand what he/she could do next. Let me tag the right SME here?